openapi_python_client/templates/client.py.jinja

```diff
@@ -71,6 +75,15 @@ class Client:
         if self._async_client is not None:
             self._async_client.timeout = timeout
         return evolve(self, timeout=timeout)
+
+    def with_retries(self, retries: int) -> "{{ self }}":
+        """Get a new client matching this one with a retry"""
+        self._transport = httpx.HTTPTransport(retries=retries)
+        if self._client is not None:
+            self._client.transport = self._transport
+        if self._async_client is not None:
+            self._async_client.transport = self._transport
```

Comment:
Just reading through the httpx code, I don't think you can set transport like that. I'm pretty sure you can only set transport when first constructing the inner clients. So I don't think `with_retries` would work for this one.